packages/xpetra/src/MultiVector/Xpetra_MultiVectorFactory_decl.hpp

```diff
@@ -58,7 +58,7 @@
 #include "Xpetra_EpetraIntMultiVector.hpp"
 #endif
 
-#include "Xpetra_BlockedMap.hpp"
+// #include "Xpetra_BlockedMap.hpp"
```

Comment:
Here are the rules for whether or not to include this header:

1. If you use the class by pointer, you only need a forward declaration.
2. If you use a typedef in the class, you need an actual class declaration.
3. If you call a method in the class, you need the class' definition.

It looks like `Xpetra::BlockedMap` does not show up in this file at all, so commenting it out is right.